tests/command/core_test.py

```diff
@@ -0,0 +1,49 @@
+"""Test the core event handler."""
+from unittest import mock
+from command.core import Core
+
+
+def test_handle_invalid_mention():
+    """Test the instance of handle_app_mention being called inappropriately."""
+    event = {}
```

Comment:
This can be made a bit cleaner using a dictionary literal, which would something look like this:

```python
event = {
    "token": "XXYYZZ",
    "team_id": "TXXXXXXXX",
    "api_app_id": "AXXXXXXX",
    "event": {
        "type": "app_mention",
        ...,
    },
    ...
}
```

Reply:
Ok cool, I'll write it like that